src/django/api/reports.py

```diff
@@ -120,6 +121,102 @@ def weekly_geocoding_time_with_queue():
     return [['week', 'average_geocoding_time_in_seconds'], rows]
 
 
+def submitted_product_type_values():
+    data = defaultdict(int)
+
+    product_type_lists = ExtendedField.objects.filter(
+        field_name='product_type'
+    ).values_list('value__raw_values', flat=True)
+    for product_types in product_type_lists:
+        for p in product_types:
+            data[p.strip()] = data[p.strip()] + 1
+    rows = sort_by_first_column(data.items())
+    return [['product_type_values', 'times_submitted'], rows]
+
+
+def processing_type_facility_type_matched():
+    temp_data = defaultdict(set)
+
+    for (raw_values, matched_values) in ExtendedField.objects.filter(
+        Q(field_name='processing_type') | Q(field_name='facility_type')
+    ).distinct(
+        'value__raw_values', 'value__matched_values'
+    ).values_list('value__raw_values', 'value__matched_values').iterator():
+        for i, raw_value in enumerate(raw_values.split('|')):
+            if matched_values[i][3] is not None:
+                temp_data[matched_values[i][3]].add(raw_value.strip())
+                temp_data[matched_values[i][2]].add(raw_value.strip())
+
+    data = dict()
+    for (raw_value, match_values) in temp_data.items():
+        data[raw_value] = ' | '.join(list(match_values))
+
+    rows = sort_by_first_column(data.items())
+    return [['taxonomy_value', 'raw_values'], rows]
+
+
+def processing_type_facility_type_unmatched():
+    data = defaultdict(int)
+
+    for (raw_values, matched_values) in ExtendedField.objects.filter(
+        Q(field_name='processing_type') | Q(field_name='facility_type')
+    ).distinct(
+        'value__raw_values', 'value__matched_values'
+    ).values_list('value__raw_values', 'value__matched_values').iterator():
+        for i, raw_value in enumerate(raw_values.split('|')):
+            if matched_values[i][3] is None:
+                data[raw_value.strip()] = data[raw_value.strip()] + 1
+
+    rows = sort_by_first_column(data.items())
+    return [['processing_type_facility_type', 'times_submitted'], rows]
+
+
+def contributors_with_extended_fields_cumulative():
+    data = defaultdict(list)
+    months = ExtendedField.objects.dates('updated_at', 'month')
+    for month in months:
+        m = month.strftime(DateFormats.MONTH)
+        data[m] = [m]
+        fields = ExtendedField.objects.filter(
+            updated_at__year__lte=month.year,
+            updated_at__month__lte=month.month)
+
+        for field_name, _ in ExtendedField.FIELD_CHOICES:
```

Comment:
Not a problem introduced in this PR, but adding this use of `FIELD_CHOICES` exposed the fact that facility type and processing type are repeated . We will want to remove them, which will likely require creating a migration since is "changes" the choices attached to a model field.

<img width="1100" alt="166006960-ad98c044-0077-4889-b8cf-7c31f04cc1a9" src="https://user-images.githubusercontent.com/17363/166078148-9ec17355-fc44-4f78-ae13-0290049292b1.png">

https://github.com/open-apparel-registry/open-apparel-registry/blame/a070e8be9dce56112de4ebb757f477c374b0e595/src/django/api/models.py#L2387-L2398

Reply:
I made a model change and a migration in [2ae5f45](https://github.com/open-apparel-registry/open-apparel-registry/pull/1826/commits/2ae5f4525a8119eaf3b172fd9f4c12168fcc1e7a). I checked my local data after migrating to confirm that removing two of the values didn't have any impact on my processing_type and facility_type fields, just in case, and confirmed there were no empty values for the field_name.